scalableminds/webknossos
# Add manual pass state for long-running jobs

Shows `convert_to_wkw` jobs that failed as “Manual“ (meaning an admin will try to repair this job), unless an admin has alreadyset the job’s `manualState` to `SUCCESS` or `FAILURE` in the database.

### Steps to test:
- set up worker, jobsEnabled
- upload datasets that work and that don’t work
- run other jobs as well
- in postgres, set a job to manually repaired or failure (i.e. wontfix)
- that state should be shown then, links should become available in success case

### Note:

I am unsure about the wording (e.g. the displayed name “Manual” for that new state), and the tooltips. Feedback welcome :)

### Issues:
- fixes #5516 
- fixes #5466 

------
